wardemo/src/main/java/org/jfclere/tomcatPI/PIFrameBuffer.java

```diff
@@ -59,63 +58,11 @@ public int getGreen(short pixel) {
 		cache = 0x0007;
 		return (pixel & cache) * 8 + green;
 	}
-
+	
```

Comment:
If you remove the code for main() why you didn't remove it?
We don't need it correct?

Reply:
Was not sure about it, we can remove it !

Reply:
OK Cool remove it and update the PR.